src/Sylius/Bundle/UserBundle/Command/ChangePasswordCommand.php

```diff
@@ -0,0 +1,132 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sylius\Bundle\UserBundle\Command;
+
+use Doctrine\ORM\EntityManagerInterface;
```

Comment:
Typehinting and using `ObjectManager` as interface should be enough here, we don't use any methods specific to entity manager.